open-api/NOTICE

```diff
@@ -0,0 +1,26 @@
+
+Apache Iceberg
+Copyright 2017-2024 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+
+This project includes code from Kite, developed at Cloudera, Inc. with
+the following copyright notice:
+
+| Copyright 2013 Cloudera Inc.
```

Comment:
Do we ship anything from Cloudera here ? I'm a bit surprised.

Reply:
we do have some code form it I guess. 

More content: https://github.com/apache/iceberg/pull/11279#discussion_r1805082199

Reply:
The good ol' schema visitor: https://github.com/apache/iceberg/blob/9e895cb6dff9dcd2a117a4f5e197f0235047ff54/LICENSE#L239-L245